app/migrations/0029_rename_default_odm_node.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+from nodeodm.models import ProcessingNode
+
+def rename_default_node(apps, schema_editor):
```

Comment:
Docker names the nodes `webodm_node-odm_X`. So I'm migrating previously default nodes to the new name.